docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,27 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
+
+The component build pipeline for {ProductName} include various types of build-time tests, conducted using link:https://tekton.dev/docs/pipelines/tasks/#overview[Tekton tasks]. For validating container information, the pipeline employs https://www.conftest.dev/[Conftest]. The table below outlines the currently implemented build-time tests:
+
+.Deprecated image checks
+|===
+|Test name |Description |Failure message
+
+|image_repository_deprecated |Deprecated images are no longer maintained, leading to unresolved security vulnerabilities. | The container image must not be built from a repository  marked as 'Deprecated' in COMET
+|===
+
+.Unsigned RPM check
+|===
+|Test name |Description |Failure message
+
+|image_unsigned_rpms |Packages signed with Red Hat's secure signing server adheres to stringent policies and procedures. |All RPMs in the image must be signed. Found following unsigned rpms(nvra):
```

Comment:
The test doesn't enforce so it is hard to say something like "all RPMs in the image must be signed" instead, it may be better to say that it alerts if there are any unsigned RPMs included.

Does it make sense to change the first sentence as well to add the relevant perspective for this use case without assuming that users are within Red Hat?